### PR TITLE
Updated docs for installing kritis

### LINKED ISSUE #79: Update README.md to include additional kritis info for users
Update the README.md to include:

- [ ] kritis install/delete information (certgen, container-analysis, pre-install, helm, cleanup, updating) 
- [ ] kritis on GKE install guide
- [ ] kritis use example (crd, isp, attestation, etc)
- [ ] link to grafeas.io kritis whitepaper

### PR DESCRIPTION
**Pull request by priyawadhwa**
Partially fixes #79, still need to add the use example

### CODE REVIEW

File: kritis-charts/README.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+To create the secret:
+1. Create a service account with `Container Analysis Admin` permissions in the Google Cloud Console project that hosts the images kritis will be inspecting
+2. Download a JSON key for the service account
+3. Rename the key to `kritis.json`
+4. Create a Kubernetes secret by running:
+```
+kubectl create secret generic <SECRET NAME> --from-file=<path to kritis.json>
+```
+
 ## Installing Kritis
```
Comment (tejal29) [inline kritis-charts/README.md]: Can you copy the command from kritis-charts/ here and update to use helm certgen plugin

**Comment by tejal29** [comment]
We also need to add Tutorial with sample specs and  deployments.

**Comment by priyawadhwa** [comment]
for sure, I'll open a separate PR with the tutorial

**Comment by tejal29** [comment]
Here is the issue for it #87

File: kritis-charts/README.md (hunk 1 above)
Comment (priyawadhwa) [reply kritis-charts/README.md]: done

File: kritis-charts/README.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+### Enabling the Container Analysis API
+You will need to enable the Container Analysis API and enable Vulnerability Scanning in your Google Cloud Console project.
+Instructions can be found in the `Before you Begin` section of the [Getting Image Vulnerabilities](https://cloud.google.com/container-registry/docs/get-image-vulnerabilities#before_you_begin) docs.
+kritis can only inspect images hosted in projects that have both of these enabled, and have already been scanned for vulnerabilities.
+
+### Creating a Container Analysis Secret
+You will need to create a Kubernetes secret, which will provide kritis with the auth required to get vulnerability information for images.
+To create the secret:
+1. Create a service account with `Container Analysis Admin` permissions in the Google Cloud Console project that hosts the images kritis will be inspecting
+2. Download a JSON key for the service account
```
[the remark this answers is not in the document]
Comment (priyawadhwa) [reply kritis-charts/README.md]: added this

**Pull request merged**